docker/README.md

```diff
@@ -4,12 +4,6 @@ You can either build the images locally, or get the images from the Docker hub.
 
 ## Download images from the Docker hub
 
-### Fedora images
-
-    docker pull fedoraqa/openqa_data
-    docker pull fedoraqa/openqa_webui
-    docker pull fedoraqa/openqa_worker
-
```

Comment:
I see that you mentioned "Also remove the outdated Fedora guide to launch the web UI via Docker" in your git commit messages but as these images are still there on docker hub we should be nice and check with the providers of these images before removing the reference here. Also I recommend you keep the Fedora firewall rules unless explicitly stated why they should be removed

Reply:
WIP trying to figure out who is the provider for these images

Reply:
I keeped the fedora section